libs/designer/src/lib/ui/panel/configureTemplatePanel/configureWorkflowsPanel/tabs/customizeWorkflowsTab.tsx

```diff
@@ -1,51 +1,206 @@
 import type { AppDispatch } from '../../../../../core/state/templates/store';
 import constants from '../../../../../common/constants';
-import type { TemplateTabProps } from '@microsoft/designer-ui';
+import { TemplatesSection, type TemplateTabProps, type TemplatesSectionItem } from '@microsoft/designer-ui';
 import { closePanel } from '../../../../../core/state/templates/panelSlice';
 import type { ConfigureWorkflowsTabProps } from '../configureWorkflowsPanel';
 import type { IntlShape } from 'react-intl';
-import { Text } from '@fluentui/react-components';
-import { initializeWorkflowsData } from '../../../../../core/actions/bjsworkflow/configuretemplate';
+import type { WorkflowTemplateData } from '../../../../../core';
+import { Accordion, AccordionHeader, AccordionItem, AccordionPanel, Text } from '@fluentui/react-components';
+import type { Template } from '@microsoft/logic-apps-shared';
+import { useMemo } from 'react';
+import { useTemplatesStrings } from '../../../../templates/templatesStrings';
+import { getWorkflownameFromWorkflowId } from '../../../../../core/actions/bjsworkflow/configuretemplate';
 
-export const CustomizeWorkflows = () => {
-  return <Text>CustomizeWorkflows</Text>;
```

Comment:
Put tab related components in their own folders, like workflows inside workflows folder inside configuretemplate so it is oprganized correctly, since we will have multiple components and if the file is becoming too large lets separate out the components in different files.